### PR TITLE
Fixes: #1811: Fix for a few issues in aws-cleanup

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

**Comment by ebenahar** [comment]
Verified by https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-aws-cleanup/754/consoleFull

**Comment by ebenahar** [comment]
Another run:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-aws-cleanup/771/console

File: tests/conftest.py [hunk 1]
```diff
@@ -910,6 +912,7 @@ def tier_marks_name():
 @pytest.fixture(scope='function', autouse=True)
 def health_checker(request, tier_marks_name):
     node = request.node
+    request.addfinalizer(ceph_health_check)
```
Comment (RazTamir) [inline tests/conftest.py:915]: this isn't up to date

Comment (ebenahar) [reply tests/conftest.py:915]: sorry, added by mistake :)

**Pull request merged**